src/auction.js

```diff
@@ -451,7 +451,7 @@ export function auctionCallbacks(auctionDone, auctionInstance, {index = auctionM
   function acceptBidResponse(adUnitCode, bid) {
     handleBidResponse(adUnitCode, bid, (done) => {
       let bidResponse = getPreparedBidForAuction(bid);
-
+      events.emit(CONSTANTS.EVENTS.BEFORE_ADD_BID_RESPONSE, bidResponse);
```

Comment:
`before add bid response` seems to me like a confusing name. How about `BID_ACCEPTED` ? It follows a consistent naming pattern (BID_REJECTED, BID_TIMEOUT, etc)

Reply:
Just made the requested change